src/components/editions/headline/headline.stories.tsx

```diff
@@ -0,0 +1,127 @@
+// ----- Imports ----- //
+
+import { Display, Pillar } from '@guardian/types';
+import { boolean, withKnobs } from '@storybook/addon-knobs';
+import {
+	analysis,
+	article,
+	comment,
+	feature,
+	interview,
+	media,
+	review,
+} from 'fixtures/item';
+import type { ReactElement } from 'react';
+import { selectPillar } from 'storybookHelpers';
+import Headline from '.';
+
+// ----- Setup ------ //
+
+const isImmersive = (): { display: Display } => {
+	return {
+		display: boolean('Immersive', false)
+			? Display.Immersive
+			: Display.Standard,
+	};
+};
+
+// ----- Stories ----- //
+
+const Default = (): ReactElement => (
+	<Headline
+		item={{
+			...article,
+			...isImmersive(),
+			theme: selectPillar(Pillar.News),
+		}}
+	/>
+);
+
+const Analysis = (): ReactElement => (
+	<Headline
+		item={{
+			...analysis,
+			...isImmersive(),
+			theme: selectPillar(Pillar.News),
+		}}
+	/>
+);
+
+const Feature = (): ReactElement => (
+	<Headline
+		item={{
+			...feature,
+			...isImmersive(),
+			theme: selectPillar(Pillar.News),
+		}}
+	/>
+);
+
+const Review = (): ReactElement => (
+	<Headline
+		item={{
+			...review,
+			...isImmersive(),
```

Comment:
Don't think it's related to this PR but can a Review be immersive? Seems like the star ratings are broken in immersive.

Reply:
Thats a good question - I think not. I'll remove it to avoid any confusion.